### DISCUSSION TITLE
How to select functions, classes, blocks...?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#6510] [Feature Requests]
I know there are "editor::SelectLargerSyntaxNode" and "editor::SelectSmallerSyntaxNode" commands. Is currently selecting a function, class or a block (or even select region between current position and last position) directly supported now? 

Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
